tests/fixtures.py

```diff
@@ -1,14 +1,38 @@
 from datetime import datetime
+from pytz import UTC
+from uuid import uuid4
 
-from mlflow.entities import Metric, Param, RunTag
+from mlflow.entities import (
+    Experiment as MLExperiment,
+    LifecycleStage,
+    Metric,
+    Param,
+    Run,
+    RunTag,
+    RunData,
+    RunInfo,
+    RunStatus,
+)
 from faculty.clients.experiment import (
+    Experiment,
+    ExperimentRun,
+    ExperimentRunStatus,
     Metric as FacultyMetric,
     Param as FacultyParam,
     Tag as FacultyTag,
 )
+from mlflow_faculty.py23 import to_timestamp
 
-from pytz import UTC
 
+PROJECT_ID = uuid4()
+STORE_URI = "faculty:{}".format(PROJECT_ID)
+
+EXPERIMENT_ID = 12
+CREATED_AT = datetime.now(tz=UTC)
```

Comment:
AFAICT this (and LAST_CREATED_AT below) are not actually used. Can we remove?

Reply:
👍